CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## v0.17 [2023-07-13]
+
+_Bug fixes_
+
+- Fixed the `iam_user_uses_corporate_login_credentials` query to ensure the use of corporate login credentials instead of personal accounts. ([#97](https://github.com/turbot/steampipe-mod-gcp-compliance/pull/97))
```

Comment:
```suggestion
- Fixed the `iam_user_uses_corporate_login_credentials` query to return `info` status, when plugin authentication mechanism does not include organization viewer permission, instead of false positives. ([#97](https://github.com/turbot/steampipe-mod-gcp-compliance/pull/97))
```